client/src/components/SystemStatus/SystemStatus.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect } from 'react';
+import { Route, Switch, Link, useHistory, useLocation } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { LoadingSpinner, Section, SectionTableWrapper } from '_common';
+import * as ROUTES from '../../constants/routes';
+import { Sidebar, SectionMessage } from '_common';
+import Sysmon from '../SystemMonitor';
+import { SystemStatusQueueTable } from './SystemStatusQueueTable/SystemStatusQueueTable';
+
+import './SystemStatus.global.css';
+import styles from './SystemStatus.module.scss';
+
+const root = `${ROUTES.WORKBENCH}${ROUTES.SYSTEM_STATUS}`;
+
+const SystemStatusSidebar = ({ systemList }) => {
+  var sidebarItems = [];
+
+  systemList.forEach((system) => {
+    sidebarItems.push({
+      to: `${root}/${system.hostname}`,
+      label: `${system.display_name}`,
+      disabled: false,
+      hidden: false,
+    });
+  });
+
+  return <Sidebar sidebarItems={sidebarItems} />;
+};
+
+const Layout = ({ hostname }) => {
+  const {
+    loading,
+    error: loadingError,
+    list: systemList,
+  } = useSelector((state) => state.systemMonitor);
+
+  const selectedSystem = systemList.find((sys) => sys.hostname === hostname);
+
+  if (loadingError) {
+    return (
+      <Section className={styles['no-results-message']}>
+        <SectionMessage type="info">
+          Unable to gather system information
+        </SectionMessage>
+      </Section>
+    );
+  }
+
+  return (
+    <>
+      <Section
+        className={styles['root']}
+        contentLayoutName="twoColumn"
+        contentClassName={styles['layout']}
+        content={
+          <div className={styles['panel-1']}>
+            <Sysmon system={selectedSystem?.hostname} />
+            {loading ? (
+              <LoadingSpinner />
+            ) : (
+              <SectionTableWrapper
+                className={styles['content']}
+                contentShouldScroll
+              >
```

Comment:
```suggestion
              <SectionTableWrapper contentShouldScroll>
```

**Optional.** The `content` class is unused, because it does not have any styles in `SystemStatus.module.css`. It can remain as an expected and available class name, or be removed so no one thinks it has styles.